content/en/news/security/istio-security-2021-003/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: ISTIO-SECURITY-2021-003
+subtitle: Security Bulletin
+description: Potential misuse of mTLS-only fields in AuthorizationPolicy with plain text traffic. 
+cves: [N/A]
+cvss: "N/A"
+vector: ""
+releases: ["All releases 1.5 and later"]
+publishdate: 2021-04-15
+keywords: [CVE]
+skip_seealso: true
+---
+
+{{< security_bulletin >}}
+
+This is a security advisory for customers to check the authorization policy to make sure [mTLS (STRICT mode) is enabled](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode)
+when using [mTLS-only fields](/docs/concepts/security/#dependency-on-mutual-tls) in the authorization policy.
+
+You can stop reading if
+
+- Your authorization policy does not use [mTLS-only fields](/docs/concepts/security/#dependency-on-mutual-tls); or
+
+- Your authorization policy uses mTLS-only fields and you have also enabled mTLS with STRICT mode or your authorization
+policy is configured to reject plain text traffic explicitly.
+
+## Issue
+
+In authorization policy, the following are mTLS-only fields:
```

Comment:
In the first paragraph we have a links for mTLS-only fields, which also has an additional field. Maybe that's fine, but wanted to call it out here as a difference. Would it make sense to create a link within this document to the fields (add link here), and use that above?

Reply:
Good point, actually I think the additional field `connection.sni` should be removed from the page as it does not depend on mutual TLS (it only requires TLS).